### PR TITLE
Added two new themes

### PR DESCRIPTION
**Pull request by Konrad77**
Two new themes. Mono (Monochrome) and PurpleRiot,  a nice looking purple theme.

### CODE REVIEW

File: data/styles/mono.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
+# based on Themer default color scheme
+# Copyright (c) 2018 Mikael Konradsson
+# with tweaks for languages other than C++ by Kacper Kasper
```
Comment (KapiX) [inline data/styles/mono.yaml]: Is this and first line still accurate?

Comment (Konrad77) [reply data/styles/mono.yaml]: No, I made this up my self.

**Comment by Konrad77** [comment]
Uploaded a new version without the text. Thanks for noticing.

**Comment by KapiX** [comment]
Thanks!

**Pull request merged**